typescript/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@revenuecat/purchases-hybrid-typescript-interfaces",
```

Comment:
what about just?

```
@revenuecat/purchases-typescript
```

In case we want to add something that are not interfaces. The `hybrid` part I think is too generic and I wish I hadn't added it to `purchases-hybrid-common` lol

Reply:
Hmm I do wonder if it's too generic and might lead some people to confuse it and use it (note that this is meant to be internal... I guess we could call it `@revenuecat/purchases-typescript-internal` maybe? wdyt?

Reply:
I like `@revenuecat/purchases-typescript-internal` yes!